app/src/RoomClient.js

```diff
@@ -3965,11 +4002,21 @@ export default class RoomClient
 
 				let producer;
 
+				const networkPriority =
+					window.config.networkPriorities.extraVideo ?
+						window.config.networkPriorities.extraVideo :
+						DEFAULT_NETWORK_PRIORITIES.extraVideo;
+
 				if (this._useSimulcast)
 				{
 					const encodings = this._getEncodings(width, height);
 					const resolutionScalings = getResolutionScalings(encodings);
 
+					encodings.forEach((encoding) =>
+					{
+						encoding.networkPriority=networkPriority;
+					});
```

Comment:
@misi maybe should we add the same fix and comment here?

Reply:
Great catch thanks! I will i will fix it.